route/mnp.js

```diff
@@ -411,7 +412,7 @@ function renderTeam(params) {
     canEdit: perms.canEdit,
     canRemove: perms.canRemove,
     canBegin: !team.ready && match.state == CONST.PREGAME && perms.canEdit && team.lineup.length > 7,
-    sugs: match.state == CONST.PREGAME ? JSON.stringify(players.getSuggestions(),null,2) : '[]'
+    sugs: match.state == CONST.PREGAME ? JSON.stringify(getSuggestions([match.home.key, match.away.key]),null,2) : '[]'
```

Comment:
Since the only team that should be editing a lineup is the team itself, we can probably just pass in a single team to exclude on. Perhaps setup getSuggestions to accept array or string.